frontend/src/static/js/v_summary.js

```diff
@@ -125,16 +125,20 @@ window.vSummary = {
     },
     sortGroupSelection() {
       this.getFiltered();
+      deactivateAllOverlays();
     },
     sortWithinGroupSelection() {
       this.getFiltered();
+      deactivateAllOverlays();
     },
     filterTimeFrame() {
       this.getFiltered();
+      deactivateAllOverlays();
```

Comment:
This might not be necessary as we are planning to make the zoom tab reactive towards `filterTimeFrame`

Reply:
Correct, this is not necessary if we are going to make it responsive to zoom.